### ISSUE TITLE
About Page word wrapping and add contact links

### ISSUE THREAD

**Issue by joshuaogle** [andreweick/aedc2#9]
Screenshot: http://www.flickr.com/photos/andreweick/11674724836

**Comment by andreweick** [comment]
For what it's worth, this was the old code

```
  <ul>
    <li>
      <a class="ss-home" href="/">Home</a>
    </li>
    <li>
      <a class="ss-contacts" href="/about/">About</a>
    </li>
    <li>
      <a class="ss-userframe" href="http://eick.us/portfolio/">Portfolio</a>
    </li>
    <li>
      <a class="ss-mailbox" href="/contact/">Contact</a>
    </li>
    <li>
      <a class="ss-buildings" href="http://missionfocus.com">My Company</a>
    </li>
    <li>
      <a class="ss-bank" href="http://imintel.org">My Institute</a>
    </li>
  </ul>
  <ul>
    <li>
      <a class="ss-social-circle ss-social-twitter" href="http://twitter.com/andreweick">Twitter</a>
    </li>
    <li>
      <a class="icon-appnet" href="http://alpha.app.net/eick">App.net</a>
    </li>
    <li>
      <a class="ss-social-circle ss-social-octocat" href="http://github.com/andreweick">Github</a>
    </li>
    <li>
      <a class="ss-social-circle ss-social-googleplus" href="https://plus.google.com/109723980772979274704/about">Google+</a>
    </li>
    <li>
      <a class="ss-social-circle ss-social-linkedin" href="http://linkedin.com/in/andreweick">LinkedIn</a>
    </li>
  </ul>
</div>

```

**Comment by joshuaogle** [comment]
The About page has been updated to handle the responsive size better and I've added the social links.

I'm not exactly sure why the old site had the duplication of the primary nav, so I've left that out. It's a lot simpler and more attractive, but if there was a compelling reason for having it there let me know.

Thanks again for the markup, that helped make this quick.

Desktop:
![screen shot 2014-01-14 at 11 52 29 am](https://f.cloud.github.com/assets/111194/1913171/1e341ff0-7d45-11e3-92aa-3bd417325c59.png)

Mobile:
![screen shot 2014-01-14 at 11 52 18 am](https://f.cloud.github.com/assets/111194/1913174/2427178c-7d45-11e3-816f-acbfb7bfa475.png)
